ko/README.md

```diff
@@ -56,6 +56,7 @@ The following generators are delivered by independent projects (and supported by
 |:------------------------- |:------------------------------------------------------------- |:----------:|:-------------------------------:|:----------------------------------- |:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | C                         | [fastMavlink](https://github.com/olliw42/fastmavlink)         |  &check;   |             &check;             | &cross;                             | Highly efficient C library with python code generators. Has [docs](https://github.com/olliw42/fastmavlink), [examples](https://github.com/olliw42/fastmavlink/tree/master/examples), [test](https://github.com/olliw42/fastmavlink/tree/master/tests), support for [routing](https://github.com/olliw42/fastmavlink#router) and [mavgen mimicry](https://github.com/olliw42/fastmavlink#pymavlink-mavgen-mimicry). |
 | Clojure                   | [clj-mavlink](https://github.com/WickedShell/clj-mavlink)     |  &check;   |             &check;             | &check;                             | Clojure MAVLink Bindings.                                                                                                                                                                                                                                                                                                                                                                                          |
+| Dart                      | [dart_mavlink](https://github.com/nus/dart_mavlink)           | &check;    |             &check;             | &cross;                             | MAVLink library for Dart.                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
Please remove this. Translations get updated automatically with links in the en/ version

Reply:
I have removed this line.